src/modules/cluster/cluster-list/cluster-list.js

```diff
@@ -27,17 +27,14 @@
         startTimer();
```

Comment:
@cloudbehl StartTimer should not be started here as you are already starting it inside init().